components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenMgtDAO.java

```diff
@@ -3141,7 +3142,7 @@ private static int getTokenPersistPoolSize () {
             }
         } catch (NumberFormatException e) {
             if (log.isDebugEnabled()) {
-                log.debug("Error while parsing OAuth Token Persistence PoolSize", e);
+                log.debug("Error while parsing OAuth Token Persistence PoolSize" + maxPoolSize, e);
```

Comment:
A NumberFormatException can occur when doing the `Integer.parseInt(maxPoolSizeConfigValue)` operation. So `maxPoolSize` will print DEFAULT_POOL_SIZE as it's the value assigned initially. Then the debug log will give an incorrect information. So it's better to add `maxPoolSizeConfigValue` for the log since it's the value that caused the NumberFormatException. WDYT?

Reply:
Done! Fix with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/472